### PR TITLE
apm: document minimum support for span links in client libraries

### PR DESCRIPTION
**Pull request by mabdinur**
### What does this PR do? What is the motivation?

Documents support for span links across apm client libraries

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 1]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
...
+## Minimum Support
+
+``datadog-agent>=7.52.0`` is required to generate Span Links using datadog client libraries (ex: ddtrace). Span Links are supported in the following versions of datadog client libraries:
+
+- Python: ddtrace>=2.5.0
+- GO: ddtrace>=1.61.0
+- PHP: .....
+- Java: ....
+- Node: ...
+- .NET: ...
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:60]:
```suggestion
- PHP:  v0.97.0
- Java:  v1.26.0
- Node: v5.3.0
- .NET: WIP
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 2]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
 - The OpenTelemetry SDK, follow the OpenTelemetry manual instrumentation documentation for your language. For example, [Create spans with links for Java][3].
 - The PHP Datadog library, follow the [Adding span links][1] examples.
 
+## Minimum Support
+
+``datadog-agent>=7.52.0`` is required to generate Span Links using datadog client libraries (ex: ddtrace). Span Links are supported in the following versions of datadog client libraries:
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:53]:
```suggestion
``datadog-agent>=7.52.0`` is required to generate Span Links using datadog client libraries (ex: ddtrace). Support for span links was introduced in the following releases of datadog apm client libraries:
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 3]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
...
+
+- Python: ddtrace>=2.5.0
+- GO: ddtrace>=1.61.0
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:56]:
```suggestion
- Python: 2.5.0
- GO:  1.61.0
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 4]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
 - The OpenTelemetry SDK, follow the OpenTelemetry manual instrumentation documentation for your language. For example, [Create spans with links for Java][3].
 - The PHP Datadog library, follow the [Adding span links][1] examples.
 
+## Minimum Support
+
+``datadog-agent>=7.52.0`` is required to generate Span Links using datadog client libraries (ex: ddtrace). Support for span links was introduced in the following releases:
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:53]:
```suggestion
``datadog-agent>=7.52.0`` is required to generate Span Links using datadog APM Client Libraries and the Datadog implementation of the OpenTelemetry API. Support for span links was introduced in the following releases:
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 5]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
...
+
+- Python: 2.5.0
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:55]:
```suggestion
- Python: v2.5.0
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 6]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
 - The OpenTelemetry SDK, follow the OpenTelemetry manual instrumentation documentation for your language. For example, [Create spans with links for Java][3].
 - The PHP Datadog library, follow the [Adding span links][1] examples.
 
+## Minimum Support
+
+``datadog-agent>=7.52.0`` is required to generate Span Links using datadog APM Client Libraries and the Datadog implementation of the OpenTelemetry API. Support for span links was introduced in the following releases:
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:53]:
```suggestion
``datadog-agent>=7.52.0`` is required to generate Span Links using APM Client Libraries. Support for span links was introduced in the following releases:
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 7]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
...
+
+- Python: v2.5.0
+- GO: 1.61.0
+- PHP: v0.97.0
+- Java: v1.26.0
+- Node: v5.3.0
+- .NET: WIP
+
+Note: (blurb about OTLP support for Span Links)
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:62]:
```suggestion
Note: This section documents minimum support for generating span link via Datadog APM client libraries. Span links generated by the OpenTelemetry SDK are <document support>.
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 8]
```diff
@@ -48,6 +48,19 @@ If your application is instrumented with:
...
+## Minimum Support
+
+``datadog-agent>=7.52.0`` is required to generate Span Links using APM Client Libraries. Support for span links was introduced in the following releases:
+
+- Python: v2.5.0
+- GO: v1.61.0
+- PHP: v0.97.0
+- Java: v1.26.0
+- Node: v5.3.0
+- .NET: WIP
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:60]:
```suggestion
- .NET: WIP
- Ruby: No Support
- C++/Proxy: No Support
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 9]
```diff
@@ -48,6 +48,21 @@ If your application is instrumented with:
 - The OpenTelemetry SDK, follow the OpenTelemetry manual instrumentation documentation for your language. For example, [Create spans with links for Java][3].
 - The PHP Datadog library, follow the [Adding span links][1] examples.
 
+## Minimum Support
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:51]:
```suggestion
## Minimum support
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 10]
```diff
@@ -48,6 +48,21 @@ If your application is instrumented with:
...
+- Python: v2.5.0
+- GO: v1.61.0
+- PHP: v0.97.0
+- Java: v1.26.0
+- Node: v5.3.0
+- .NET: WIP
+- Ruby: No Support
+- C++/Proxy: No Support
+
+Note: This section documents minimum support for generating span link via Datadog APM client libraries. Span links generated by the OpenTelemetry SDK are <document support>.
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:64]:
```suggestion
Note: This section documents minimum support for generating span link with Datadog APM client libraries. Span links generated by the OpenTelemetry SDK are sent to Datadog through [otlp ingest](https://docs.datadoghq.com/opentelemetry/interoperability/otlp_ingest_in_the_agent).
```

**Review by jhgilbert — APPROVED** [review]
Approved, please verify that the table I added is communicating your intended meaning. 

When this is ready for merge, you can mention me here or ping the documentation Slack channel, and I'm happy to merge it for you. Thank you!

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 11]
```diff
@@ -48,6 +48,23 @@ If your application is instrumented with:
 - The OpenTelemetry SDK, follow the OpenTelemetry manual instrumentation documentation for your language. For example, [Create spans with links for Java][3].
 - The PHP Datadog library, follow the [Adding span links][1] examples.
 
+## Minimum support
+
+**Note***: This section documents minimum support for generating span links with Datadog APM client libraries. Span links generated by the OpenTelemetry SDK are sent to Datadog through [OTLP Ingest][8].
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:53]:
```suggestion
**Note***: This section documents minimum support for generating span links with Datadog APM client libraries (via the OpenTelemetry API). Span links generated by the OpenTelemetry SDK are sent to Datadog through [OTLP Ingest][8].
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 12]
```diff
@@ -48,6 +48,23 @@ If your application is instrumented with:
 - The OpenTelemetry SDK, follow the OpenTelemetry manual instrumentation documentation for your language. For example, [Create spans with links for Java][3].
 - The PHP Datadog library, follow the [Adding span links][1] examples.
 
+## Minimum support
+
+**Note***: This section documents minimum support for generating span links with Datadog APM client libraries (via the OpenTelemetry API). Span links generated by the OpenTelemetry SDK are sent to Datadog through [OTLP Ingest][8].
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:53]:
```suggestion
**Note***: This section documents minimum support for generating span links with Datadog APM client libraries (with the OpenTelemetry API). Span links generated by the OpenTelemetry SDK are sent to Datadog through [OTLP Ingest][8].
```

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 13]
```diff
@@ -48,6 +48,23 @@ If your application is instrumented with:
...
+| Language  | Minimum tracing library version |
+|-----------|---------------------------------|
+| C++/Proxy | Not supported                   |
+| Go        | 1.61.0                          |
+| Java      | 1.26.0                          |
+| .NET      | Not supported                   |
+| Node      | 5.3.0                           |
+| PHP       | 0.97.0                          |
+| Python    | 2.5.0                           |
+| Ruby      | Not supported                   |
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/span_links/_index.md:66]:
```suggestion
| C++/Proxy | Not yet supported                   |
| Go        | 1.61.0                          |
| Java      | 1.26.0                          |
| .NET      | Not yet supported                   |
| Node      | 5.3.0                           |
| PHP       | 0.97.0                          |
| Python    | 2.5.0                           |
| Ruby      | Not yet supported                   |
```

**Pull request merged**